### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by ghost** [AykutSarac/jsoncrack.com#351] [Ideas]
.

**Comment by AykutSarac** [comment]
There's [VS Code extension](https://marketplace.visualstudio.com/items?itemName=AykutSarac.jsoncrack-vscode).

**Comment by Fcoder56** [comment]
how sovle 
g++.exe: fatal error: -fuse-linker-plugin, but liblto_plugin-0.dll not found
compilation terminated
